packages/api-client/tests/test-collections.js

```diff
@@ -101,3 +124,31 @@ test('getCollections calls the callback with the expected object and returns the
 
   t.deepEqual(result, { foo: 'bar' });
 });
+
+test('updateCollection calls the callback with the expected object', async (t) => {
+  const collection = fakeCollectionFactory();
+  await t.context.collectionModel.create(collection);
+  const updatedCollection = {
+    ...collection,
+    reportToEms: false,
+  };
```

Comment:
You really don't need to create the record in the database for this test to succeed. Nor do you need to create a complex object by overriding properties of `collection`. We just need to prove that whatever object is send in the function arguments is received in the request body